lib/lambda-worker/lambda-worker.ts

```diff
@@ -141,6 +141,7 @@ export class LambdaWorker extends Construct {
       new SqsEventSource(lambdaQueue, {
         enabled: props.lambdaProps.enableQueue ?? true,
         batchSize: 1,
+        maxConcurrency: props.lambdaProps.queueMaxConcurrency,
```

Comment:
what is the default value `maxConcurrency`  gets set to if you don't specify a value?
does it need a default value i.e. `maxConcurrency: props.lambdaProps.queueMaxConcurrency ?? <some default>`, or do we only set the property if a value for `queueMaxConcurrency` is set?

Reply:
By default, maxConcurrency has no specific limit - uses up to maximum concurrency allowed by AWS.
I would add default value only if we internally know that there is some value that suits us for our use cases. Otherwise, I think that just forwarding a parameter and leaving default behavior is the best option.

Reply:
Lets not make this an optional property, lets make it a mandatory property. This is a breaking change, but will force us to enumerate the max concurrency for every use case, which is actually the right thing to do.